addons/pos_online_payment_self_order/__manifest__.py

```diff
@@ -18,6 +18,9 @@
             'pos_online_payment_self_order/static/src/**/*',
             'web/static/lib/zxing-library/zxing-library.js',
         ],
+        'web.assets_tests': [
+            'pos_online_payment_self_order/static/tests/tours/pos_online_payment_multi_table_order.js',
+        ],
```

Comment:
Note for the next reviewer: 
@aamo-odoo told me it's needed to specify the file in this case because there are 2 tour files in the module which are never getting called.